ibm/service/vpc/resource_ibm_is_backup_policy_test.go

```diff
@@ -69,6 +69,49 @@ func TestAccIBMIsBackupPolicyBasic(t *testing.T) {
 	})
 }
 
+func TestAccIBMIsBackupPolicyRemoteCopyPolicies(t *testing.T) {
+	vpcname := fmt.Sprintf("tf-vpc-%d", acctest.RandIntRange(10, 100))
+	name := fmt.Sprintf("tf-instnace-%d", acctest.RandIntRange(10, 100))
+	subnetname := fmt.Sprintf("tf-subnet-%d", acctest.RandIntRange(10, 100))
+	sshname := fmt.Sprintf("tf-ssh-%d", acctest.RandIntRange(10, 100))
+	volname := fmt.Sprintf("tf-vol-%d", acctest.RandIntRange(10, 100))
+	backupPolicyName := fmt.Sprintf("tfbakuppolicyname%d", acctest.RandIntRange(10, 100))
+	backupPolicyNameUpdate := fmt.Sprintf("tfbakuppolicyname%d", acctest.RandIntRange(10, 100))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { acc.TestAccPreCheck(t) },
+		Providers:    acc.TestAccProviders,
+		CheckDestroy: testAccCheckIBMIsBackupPolicyDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCheckIBMIsBackupPolicyConfigRemoteCopyPolicies(backupPolicyName, vpcname, subnetname, sshname, volname, name),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("ibm_is_backup_policy.is_backup_policy", "name", backupPolicyName),
+					resource.TestCheckResourceAttrSet("ibm_is_backup_policy.is_backup_policy", "remote_copy_policies.#"),
+					resource.TestCheckResourceAttrSet("ibm_is_backup_policy.is_backup_policy", "remote_copy_policies.0.delete_over_count"),
+					resource.TestCheckResourceAttrSet("ibm_is_backup_policy.is_backup_policy", "remote_copy_policies.0.encryption_key"),
+					resource.TestCheckResourceAttrSet("ibm_is_backup_policy.is_backup_policy", "remote_copy_policies.0.region"),
+				),
+			},
+			resource.TestStep{
+				Config: testAccCheckIBMIsBackupPolicyConfigRemoteCopyPolicies(backupPolicyNameUpdate, vpcname, subnetname, sshname, volname, name),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("ibm_is_backup_policy.is_backup_policy", "name", backupPolicyNameUpdate),
```

Comment:
suggested improvement. cover the path scenarios for remote_copy_policies, whatever supported.